examples/island_cluster_with_vm_router/main.tf

```diff
@@ -0,0 +1,219 @@
+resource "google_service_account" "gke-sa" {
+  account_id = "gke-sa-${var.random_suffix}"
+}
+
+resource "google_service_account" "gke-access-sa" {
+  account_id = "gke-access-sa-${var.random_suffix}"
+}
+
+resource "google_project_iam_member" "gke-dev" {
+  project = var.project_id
+  member  = "serviceAccount:${google_service_account.gke-access-sa.email}"
+  role    = "roles/container.developer"
+}
+
```

Comment:
What is this SA for?

Reply:
Remnant from some removed changes, removed this.